src/token.rs

```diff
@@ -70,6 +75,7 @@ impl Token {
 ///     .build();
 ///
 /// ```
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
The Builder probably does not need to derive serde, or does it? If not, please remove it here.

Reply:
Yeah, you're right. Removed this with new commit.